### PR TITLE
Support verifcard

### PR DESCRIPTION
**Pull request by BushraAsif**

### CODE REVIEW

File: controllers/front/showsavedcreditcards.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function initContent()
                     $savedCreditCard[] = [
                         'userID' => $result['userID'],
                         'creditCard' => $result['creditCardNumber'],
-                        'cardName' => $result['cardName'],
+                        'cardName' => $result['cardBrand'],
                         'cardBrand' => $result['cardBrand'],
```
Comment (ss-altapay-com) [inline controllers/front/showsavedcreditcards.php:41]: Is this expected to change cardName key to use cardBrand value?

File: helpers.php [hunk 2]
```diff
@@ -220,44 +220,51 @@ function updatePaymentStatus($paymentId, $paymentStatus)
...
-           . pSQL($errorText) . "', '" . time() . "')" . ' ON DUPLICATE KEY UPDATE `paymentStatus` = ' . "'" . $paymentStatus . "'";
+        "('" . $current_order->id . "', '" . pSQL($uniqueId) . "', '"
+        . pSQL($paymentId) . "', '" . pSQL($cardMask) . "', '"
+        . pSQL($cardToken) . "', '" . pSQL($cardBrand) . "', '"
+        . pSQL($cardExpiryDate) . "', '"
+        . pSQL($cardCountry) . "', '" . pSQL($paymentType) . "', '"
+        . pSQL($paymentTerminal) . "', '"
+        . pSQL($paymentStatus) . "', '" . pSQL($paymentNature) . "', '"
+        . pSQL($requireCapture) . "', '" . time()
+        . "')" . ' ON DUPLICATE KEY UPDATE `paymentStatus` = ' . "'" . $paymentStatus . "'";
```
Comment (ss-altapay-com) [inline helpers.php:267]: paymentStatus not escaped

File: controllers/front/showsavedcreditcards.php (hunk 1 above)
Comment (BushraAsif) [reply controllers/front/showsavedcreditcards.php]: I have removed this entry as we no longer use this variable in the template file.

File: helpers.php (hunk 2 above)
Comment (BushraAsif) [reply helpers.php]: This have been updated

**Review by ss-altapay-com — APPROVED** [review]

**Pull request merged**